plugins/pagerduty/src/plugin.ts

```diff
@@ -51,7 +53,45 @@ export const EntityPagerDutyCard = pagerDutyPlugin.provide(
     name: 'EntityPagerDutyCard',
     component: {
       lazy: () =>
-        import('./components/PagerDutyCard').then(m => m.PagerDutyCard),
+        import('./components/EntityPagerDutyCard').then(
+          m => m.EntityPagerDutyCard,
+        ),
+    },
+  }),
+);
+
+export const homePlugin = createPlugin({
+  id: 'pagerduty-home',
+  routes: {
+    root: rootRouteRef,
+  },
+});
+
+/** @public */
+export const PagerDutyHomepageCard = homePlugin.provide(
+  createCardExtension<PagerDutyHomepageCardProps>({
+    name: 'PagerDutyHomepageCard',
+    title: 'Pager Duty Homepage Card',
+    components: () => import('./components/PagerDutyHomepageCard'),
+    settings: {
+      schema: {
+        title: 'Pagerduty',
+        type: 'object',
+        properties: {
+          integrationKey: {
+            title: 'Pager duty integration key',
```

Comment:
```suggestion
            title: 'PagerDuty integration key',
```

(sorry for the nitpicking; it's a product name after all)

Reply:
No probs. I don't see it as nitpicking. It makes me feel safer to know that someone is actually reading the code. :)